src/chess/bitboard.h

```diff
@@ -257,6 +257,13 @@ class Move {
 
  private:
    uint16_t data_; //first bit castling, next 3 promotion, next 6 fromsquare, next 6 tosquare
```

Comment:
You can declare this `= 0;` and remove those initializations from the constructor. 

Also, according to the style guide in use, all comments must be capitalized and end in a period (or other grammatical marker).

Reply:
Done